defaults/main.yml

```
@@ -7,14 +7,16 @@ state: latest
 tailscale_authkey: ""
 # Optional command-line arguments for 'tailscale up'
 tailscale_args: ""
+# Required device tags to advertise
+tailscale_tags: []
+# Set timeout for 'tailscale up' command
+tailscale_timeout: "120s"
```

To specify, as there may be other timeouts:

```suggestion
tailscale_up_timeout: "120s"
```